### ISSUE TITLE
Dataset Splitting

### ISSUE THREAD

**Issue by MrWhite928** [echo-sky/SFIST#2]
Is the division of the training set and test set in the dataset correct? I noticed that the content of image sequence 04 in the test set differs significantly from the sequences in the test set. How do you identify such targets that have not appeared in the training set?
